### PR TITLE
Fix spacing issues to get this branch rolling again

### PR DESCRIPTION
**Pull request by pushfoo**
tl;dr merge this PR into your branch and it'll automatically update your PR addressing   `__all__` (https://github.com/pythonarcade/arcade/pull/1681)

1. Whitespace linting was broken in arcade for months. Sorry about that, I know it created some headaches.
2. I fixed it in https://github.com/pythonarcade/arcade/pull/1706, and running `ruff` should now work for any branch up to date with development
3. I fixed up your PR directly to spare you the hassle of back and forth reviews since it's not fair for beginners to expect them to deal with broken tooling. 

`ruff --fix path/to/file/` should now auto-fix many classes of reported issues. Also, you will still get credit for the final squashed commit in arcade's git history.

### CODE REVIEW

File: arcade/gui/examples/widget_gallery.py [hunk 1]
```diff
@@ -147,9 +147,9 @@ def __init__(self):
 
         example_text = dedent(
             """
```
Comment (gran4) [inline arcade/gui/examples/widget_gallery.py:149]: Remove indent

**Review by gran4 — COMMENTED** [review]
Just that one thing

File: arcade/gui/examples/widget_gallery.py (hunk 1 above)
Comment (gran4) [reply arcade/gui/examples/widget_gallery.py:149]: Also, your doing a PR into my fork. How does that work? I'm new to open source contributing so I don't know what I'm doing with this lol.

Comment (gran4) [reply arcade/gui/examples/widget_gallery.py:149]: @pushfoo how do I merge this into my thing?
When I try it says: "merge: pushfoo-pushfoo_add__all__to_all - not something we can merge"

**Comment by gran4** [comment]
Thanks a lot. I was stuck

File: arcade/gui/examples/widget_gallery.py (hunk 1 above)
Comment (gran4) [reply arcade/gui/examples/widget_gallery.py:149]: > When I try it says: "merge: pushfoo-pushfoo_add__all__to_all - not something we can merge"

I found a way to do that, but scene might not be in there
